source/MRMesh/MRMesh.vcxproj.filters

```diff
@@ -1256,6 +1257,7 @@
     <ClCompile Include="MRPolyline2Collide.cpp">
       <Filter>Source Files\AABBTree</Filter>
     </ClCompile>
+    <ClCompile Include="MRObjectPointsHolder.cpp" />
```

Comment:
should it be in DataModel/ModelHolders ?

Reply:
fixed